fpdf/output.py

```diff
@@ -169,6 +170,10 @@ def __init__(self, contents):
         self.type = Name("Metadata")
         self.subtype = Name("XML")
 
+    def set_contents(self, contents):
```

Comment:
Why adding this method?

Reply:
I was trying to solve a problem but found a better solution.
I'll remove it.